### PR TITLE
add missing coverage

### PR DESCRIPTION
**Pull request by eclendenning-va**
When dev address validation is called from outside of the va network it returns `{"message":"Your IP address is not allowed"}` while sandbox works. 

Currently I have not added that check, all other checks added were validated by curls.

### CODE REVIEW

File: pingdom-checks/data-query.ping [hunk 1]
```diff
@@ -37,3 +37,19 @@ pingdom save-check \
...
+    --template simple-resource \
+    -a name=sandbox-health-alb \
+    -a host=sandbox-api.va.gov \
+    -a url="/services/fhir/v0/healthz" \
+    -a group="health-apis" \
+    -a integrationids_csv="$HEALTH_APIS_SLACK_ID";
+
+  pingdom save-check \
+    --template fhir-resource \
+    -a name=sandbox-dstu2-patient \
```
Comment (jhulbert-va) [inline pingdom-checks/data-query.ping]: Do we want to add `Condition` and `Med-Order` checks here like we have for production?

File: pingdom-checks/ssl.ping [hunk 2]
```diff
@@ -12,6 +12,12 @@ pingdom save-check \
   -a host=dev-api.va.gov \
   -a integrationids_csv="$SSL_EXPIRATION_SLACK_ID";
 
+pingdom save-check \
+  --template ssl-expiration-check \
+  -a name=ssl-dev-api \
```
Comment (jhulbert-va) [inline pingdom-checks/ssl.ping]: `ssl-sandbox-api`

File: pingdom-checks/data-query.ping (hunk 1 above)
Comment (bschofield-va) [reply pingdom-checks/data-query.ping]: we don't have them today, so I'll say not at this time.
there has been a discussion with Gabe on what the right level to monitor resources is, we may end up doing all of them, but for now, let's just mimic what exists today

**Comment by bschofield-va** [comment]
I think since this affects monitoring of production, @golavarr-va should weigh in on when it's cool to merge it.

**Comment by golavarr-va** [comment]
Want to make sure I understand the new "resend every 2 down cycles" functionality. Currently we get alerted once in Slack when the endpoint is down, and we don't get another alert until it's up. Will this update result in a new slack notification every 2 minutes while the endpoint is still down?

**Comment by bschofield-va** [comment]
Pingdom defines alerts very specifically as _emails it sends directly_.
Generically, the notifications that we get in Slack or from PagerDuty are _integrations_ in Pingdom terms.

We do not use Pingdom to send emails to individuals or teams, PagerDuty manages that for us.

We _will_ use Pingdom based email alerts to integrate with Statuspage. Basically, each status page component gets a dedicated email. I'm decreasing the delay to provide more timely updates to Statuspage, and increasing the resend rate to help mitigate a "dropped message".

image

**Pull request merged**